build_scripts/swiftlint.sh

```diff
@@ -0,0 +1,12 @@
+if [ "${CONFIGURATION}" = "Debug" ]; then
+  echo "SwiftLint started"
+  Pods/SwiftLint/swiftlint --quiet --path "${1:-.}"
```

Comment:
👍
we can also add swiftformat linting on UnitTests run going forward

Reply:
Agree. Any kind of linters/validators will pay off.